@AH-64D Apache Longbow/Addons/fza_ah64_ihadss/functions/fn_controller.sqf

```diff
@@ -67,4 +67,14 @@ if (_acBusOn && _dcBusOn) then {
 
 if (_tadsShouldBeStowed != _heli getVariable "fza_ah64_tadsStow") then {
     [_heli, "fza_ah64_tadsStow", _tadsShouldBeStowed] call fza_fnc_animSetValue;
+};
+
+private _currentTurret = _heli call fza_fnc_currentTurret;
+private _gunnnerUnit = _heli turretUnit [0];
+
+if (_currentTurret isEqualTo [0] || !(isplayer _gunnnerUnit)) then {
+    private _azimuth = _heli animationPhase "tads_tur";
+    private _elevation = _heli animationPhase "tads";
+    _heli setVariable ["fza_ah64_tadsElevation", _elevation, true];
```

Comment:
This TADS work shouldn't be in IHADSS, it should be in our targeting subsystem.

Can you move it over there?

Also, you are currently `setVariable` ing per frame. Please can you reduce this